modules/installation-azure-permissions.adoc

```diff
@@ -10,6 +10,7 @@ Your Microsoft Azure account must have the following roles for the subscription
 that you use:
```

Comment:
To help resolve customer confusion, maybe we can add some more description about the Azure USER account purpose. 
Form Patrick's Bugzilla comments:
```
This is the user account for `az login` or when browsing the web portal. The purpose of this section is to make sure that the user has the correct permissions to create the service principal which will then be used to run the installer and the cluster.
```

Reply:
acked